lab_6_pipeline/pos_frequency_pipeline.py

```diff
@@ -1,13 +1,40 @@
 """
 Implementation of POSFrequencyPipeline for score ten only.
 """
+import re
+from pathlib import Path
 from typing import Optional
 
+from core_utils.article.article import Article, ArtifactType
+from core_utils.article.io import from_meta, to_meta
+from core_utils.article.ud import extract_sentences_from_raw_conllu
+from core_utils.constants import ASSETS_PATH
+from core_utils.visualizer import visualize
+from lab_6_pipeline.pipeline import (ConlluSentence, ConlluToken,
+                                     CorpusManager, MorphologicalTokenDTO)
+
+
+class EmptyFileError(Exception):
+    """
+    File is empty
+    """
+
 
 def from_conllu(path: Path, article: Optional[Article] = None) -> Article:
     """
     Populates the Article abstraction with all information from the conllu file
     """
+    with open(path, encoding='utf-8') as f:
+        content = f.read()
+    extracted_sentences = extract_sentences_from_raw_conllu(content)
+    for sentence in extracted_sentences:
+        sentence['tokens'] = [_parse_conllu_token(token) for token in  sentence['tokens']]
+    conllu_senteces = [ConlluSentence(**sentence) for sentence in extracted_sentences]
+    if not article:
+        article_id = int(re.match(r'\d+', path.stem).group())
```

Comment:
use get_article_id_from_filepath to get id

Reply:
fixed